general-preprocessing-rainfall_noaa.ipynb

```diff
@@ -8,11 +8,11 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    grid_area_weights = iris.analysis.cartography.area_weights(Borneo)
</pre><p>What is this line actually doing? </p><p>Maybe add one more comment to clarify what is happening here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Environmental-DS-Book/general-preprocessing-rainfall_noaa/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>That's an interesting question involving some concepts of the Earth's shape/grid. Thanks to your suggestion I've added some extra description (in the text cell but not as a code comment), as follows:</p><p><em>Note that due to the spherical nature of the planet Earth, the area of every grid-box is not the same, therefore we need to perform the weighted mean based on the weights by area.</em></p>